views/js/controller/creator/encoders/dom2qti.js

```diff
@@ -137,40 +138,42 @@ define([
          * @returns {Array}
          */
         decode: function (nodeValue) {
-            var self = this;
-            var $nodeValue = (nodeValue instanceof $) ? nodeValue : $(nodeValue);
-            var result = [];
-            var nodeName;
+            const self = this;
+            const $nodeValue = (nodeValue instanceof $) ? nodeValue : $(nodeValue);
+            const result = [];
+            let nodeName;
 
             _.forEach($nodeValue, function (elt) {
-                var object;
+                let object;
                 if (elt.nodeType === 3) {
                     if (!_.isEmpty($.trim(elt.nodeValue))) {
                         result.push(qtiElementHelper.create('textRun', {
-                            'content': elt.nodeValue,
+                            'content': DOMPurify.sanitize(elt.nodeValue, {
+                                FORBID_TAGS: ['script']
```

Comment:
Do we need `FORBID_TAGS: ['script']`?
I think `script` is forbidden by default

Reply:
Yes, you are right